lib/scss/stanford_events.node.scss

```diff
@@ -1,5 +1,10 @@
 @charset 'UTF-8';
 @import "./stanford_events.config";
+@import 'components/event-list/event-date-time-icons';
+@import 'components/event-card/event-stacked-date--card';
+@import 'components/event-card/event-spacing-explore-more--card';
+@import 'components/event-card/event-card--heading-link';
+@import 'components/event-card/event-card--item-type-link';
```

Comment:
We shouldnt import everything on the event node styles. This adds unnecessary styles when there are no event lists or event cards.